tasks/views.py

```diff
@@ -65,6 +66,17 @@ def get_context_data(self, **kwargs):
         context["project"] = get_object_or_404(Project, pk=self.kwargs.get("pk"))
         return context
 
+    def post(self, request, pk):
+        if request.is_ajax():
+            data = request.POST
+            task = Task.objects.get(pk=data.get("task_id"))
+            if task.completed:
+                task.completed = False
+            else:
+                task.completed = True
```

Comment:
```suggestion
task.completed = not task.completed
```

Reply:
Ideally this should be done in Update view not in List View. Please create separate update view for task and use it